R/derive_vars_dy.R

```diff
@@ -84,29 +90,36 @@ derive_vars_dy <- function(dataset,
 
   if (length(bad_vars > 0)) {
     err_msg <-
-      sprintf(paste0("source_vars must end in DT or DTM or be explicitly and uniquely named.\n",
-                     "Please name or rename the following source_vars:\n", "%s"),
-              paste0(bad_vars, collapse = ", "))
+      sprintf(
+        paste0(
+          "source_vars must end in DT or DTM or be explicitly and uniquely named.\n",
+          "Please name or rename the following source_vars:\n", "%s"
+        ),
+        paste0(bad_vars, collapse = ", ")
+      )
     abort(err_msg)
   }
 
   dy_vars <- if_else(source_names == "",
-                     stringr::str_replace_all(vars2chr(source_vars), "(DT|DTM)$", "DY"),
-                     source_names)
+    stringr::str_replace_all(vars2chr(source_vars), "(DT|DTM)$", "DY"),
+    source_names
+  )
   warn_if_vars_exist(dataset, dy_vars)
 
   if (n_vars > 1L) {
     dataset %>%
-      mutate_at(.vars = source_vars,
-                .funs = list(temp = ~
-                  compute_duration(start_date = eval(reference_date), end_date = .))
-                ) %>%
-      rename_at(vars(ends_with("temp")),
-               ~ dy_vars)
+      mutate_at(
+        .vars = source_vars,
+        .funs = list(temp = ~
+        compute_duration(start_date = eval(reference_date), end_date = .))
+      ) %>%
+      rename_at(
+        vars(ends_with("temp")),
+        ~dy_vars
+      )
   } else {
     dataset %>%
       mutate(!!sym(dy_vars) :=
-               compute_duration(start_date = !!reference_date, end_date = !!source_vars[[1]])
-             )
+        compute_duration(start_date = !!reference_date, end_date = !!source_vars[[1]]))
```

Comment:
should `!!sym(dy_vars)` be on next line to align `mutate` parameters.

Reply:
Yes